README.md

```diff
@@ -73,11 +114,14 @@ The main configurations for this tool can be managed from the web page. Here are
 
 ### AIVOC_DATA_DIR
 
-Specifies the path where the vocabulary data files are stored. The default path is the current user's home directory: ~/.
+Specifies the path where the vocabulary data files are stored.
 
-Example:
+- For direct installation: The default path is the current user's home directory (`~/.`)
+- For Docker installation: Data is stored in the `aivoc_data` directory within your project folder
```

Comment:
A comment in the docker-compose file should suffice. I don't think it's absolutely necessary to mention the Docker installation when introducing the environment configuration.